.pre-commit-config.yaml

```diff
@@ -5,20 +5,11 @@ repos:
   - id: isort
     args: ["--profile", "black", "--line-length=79"]
 - repo: https://github.com/psf/black
-  rev: 22.6.0
+  rev: 22.3.0
   hooks:
   - id: black
 - repo: https://github.com/pre-commit/pre-commit-hooks
   rev: v2.3.0
   hooks:
   - id: flake8
-    args: ['--ignore=E203,W503'] # see https://github.com/psf/black/issues/315 https://github.com/psf/black/issues/52
-- repo: https://github.com/getindata/py-pre-commit-hooks
-  rev: v0.1.3
-  hooks:
-  - id: pyspelling-docker
```

Comment:
Why do we get rid of both hooks here?

Reply:
good catch, I'll add those to the template as well.